projects/services/storage-service-typescript/src/server.ts

```diff
@@ -6,7 +6,7 @@ import BlockStorageService from "./block-storage-service";
 import StateStorageService from "./state-storage-service";
 
 export default function(nodeTopology: any, env: any) {
-    const { NODE_NAME, NODE_ENV, BLOCK_STORAGE_POLL_INTERVAL, BLOCK_STORAGE_DB_PATH } = env;
+    const { NODE_NAME, NODE_ENV, BLOCK_STORAGE_POLL_INTERVAL, BLOCK_STORAGE_DB_PATH, STATE_STORAGE_POLL_INTERVAL } = env;
```

Comment:
Please add a description of this variable to documentation in `deploy/bootstrap/README.md`.